### DISCUSSION TITLE
Gravitons and QuantumField[]

### DISCUSSION THREAD

**Discussion by nmoynihan** [FeynCalc/feyncalc#119] [Q&A]
I often use FeynRule, together with e.g. QuantumField[GaugeField, {\[Mu]}, {a}][p1] to derive some interesting quantity from a Lagrangian, say a gluon three-particle vertex.

Is there any way I can do this with a spin-2 graviton, i.e. replace 'GaugeField' with Graviton somehow?

Many thanks,
Nathan

**Comment by vsht** [comment]
I don't think so, as `FeynRule` can deal only with fields carrying a single Lorentz index. `FeynRules` (the package)
can derive Feynman rules for gravity, though.

**Comment by huansouza** [comment]
Hey Nathan,

I don't know if this is of your interest, but I and some collaborators have been working with linearized gravity as an effective field theory and we have used FeynRules + FeynArts + FeynCalc to compute the diagrams. If this is your case, you can take a look at [2105.12732](https://arxiv.org/abs/2105.12732) and the Mathematica notebook and the models file you can find here: https://github.com/lehum/einstein-qed-2 .

Also, if you are interested in two-loop computations, we have used TARCER to compute them in the paper [2105.12577](https://arxiv.org/abs/2105.12577).

Best wishes,

Huan.

**Reply by Yong-Hua-Wang** [reply]
Could you upload the .fr model file? please
